site/content/en/docs/Reference/gameserverallocation.md

```diff
@@ -7,6 +7,10 @@ description: "A `GameServerAllocation` is used to atomically allocate a GameServ
 weight: 30
 ---
 
+{{% feature publishVersion="1.38.0" %}}
+When we have a fleet of game servers ready, allocation is the process of selecting a one of these servers for use. This is achieved using a `GameServerAllocation`. The allocation process marks the chosen game server for gameplay and marks it as `Allocated`. This ensures that the server is reserved for active players and not considered `ready` for other allocations until `SDK.Shutdown()` is called, or it is manually deleted.
```

Comment:
```suggestion
Allocation is the process of selecting the optimal `GameServer` that matches the filters defined in the `GameServerAllocation` specification below, and returning its details.

A successful Alloction moves the `GameServer` to the `Allocated` state, which demarcates that it is currently active, likely with players on it, and should not be removed until SDK.Shutdown() is called, or it is explicitly manually deleted.
```

Some tweaks to the language.

@roberthbailey, @Kalaiselvi84  WDYT?

Reply:
I've updated this change. Shall we include this info in FAQ page?

Reply:
So the answer is `no`.  Shall we merge this PR?